client/starwhale/api/_impl/dataset/builder.py

```diff
@@ -641,6 +641,9 @@ def close(self) -> None:
         self.join()
         if self._builder:
             self._builder.close()
+            # TODO: add len api for tabular_dataset to reduce overhead here
+            table_rows = [row for row in self._builder.tabular_dataset.scan()]
+            self.summary.rows = len(table_rows)
```

Comment:
move these code into https://github.com/star-whale/starwhale/blob/main/client/starwhale/api/_impl/dataset/model.py#L640?  it will update the summary info in the manifest file.

Reply:
done